.gitignore

```diff
@@ -128,4 +128,9 @@ $RECYCLE.BIN/
 # Mac desktop service store files
 .DS_Store
 
-_NCrunch*
\ No newline at end of file
+_NCrunch*
+launcher/obj/project.nuget.cache
```

Comment:
Unnecessary. Gitignore already has obj in it, You just need to delete the folder from the tree.

Reply:
removed what I added to gitignore